go/tools/microbench/microbench.go

```diff
@@ -159,23 +166,37 @@ func MicroBenchmark(cfg MicroBenchConfig) {
 
 		err = benchmark.execute(cfg.RootDir, w)
 		if err != nil {
+			// not stopping execution on error
 			fmt.Println(err.Error())
 		}
 
 		profiles := []string{profileMem, profileCPU}
 		for _, profile := range profiles {
 			err = benchmark.executeProfile(cfg.RootDir, profile, w)
 			if err != nil && err.Error() != errorInvalidProfileType {
+				// not stopping execution on error
 				fmt.Println(err.Error())
 			}
 		}
 		fmt.Println()
 	}
+	return nil
 }
 
-func findBenchmarks(loaded []*packages.Package) []benchmark {
-	var benchmarks []benchmark
+func findBenchmarks(loaded []*packages.Package) (benchmarks []benchmark, errs []error) {
```

Comment:
you can return a single error from here itself and does not need to concatenate outside.